lib/src/models/photo.dart

```diff
@@ -0,0 +1,52 @@
+/// A shared photo.
+class Photo {
+  Photo({
+    required this.id,
+    required this.title,
+    this.description,
+    required this.url,
+    required this.thumbnailUrl,
+    required this.createdDate,
+  });
+
+  /// Creates a [Photo] from a JSON object.
+  Photo.fromJson(Map<String, dynamic> json)
+      : this(
+          id: json['id'],
+          title: json['title'] ?? '',
```

Comment:
Though it shouldn't be allowed, there are some shared photos that don't have a `title`.